webapp/channels/src/components/admin_console/team_channel_settings/group/index.ts

```diff
@@ -18,16 +20,20 @@ type OwnProps = {
 function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
     return {
         data: ownProps.groups,
+        onPageChangedCallback: ownProps.onPageChangedCallback,
         total: ownProps.totalGroups,
         emptyListTextId: ownProps.isModeSync ? t('admin.team_channel_settings.group_list.no-synced-groups') : t('admin.team_channel_settings.group_list.no-groups'),
         emptyListTextDefaultMessage: ownProps.isModeSync ? 'At least one group must be specified' : 'No groups specified yet',
         removeGroup: ownProps.onGroupRemoved,
         setNewGroupRole: ownProps.setNewGroupRole,
+        actions: {
+            getData: () => Promise.resolve(ownProps.groups),
+        },
```

Comment:
Passing `ownProps.groups` to `Promise.resolve` for `getData()` action  pretty much solved most of the issues